spring-kafka/src/main/kotlin/io/lsdconsulting/lsd/distributed/interceptor/interceptor/ConsumerFactoryCustomizer.kt

```diff
@@ -0,0 +1,21 @@
+package io.lsdconsulting.lsd.distributed.interceptor.interceptor
+
+import org.apache.kafka.clients.producer.ProducerConfig.INTERCEPTOR_CLASSES_CONFIG
+import org.springframework.boot.autoconfigure.kafka.DefaultKafkaConsumerFactoryCustomizer
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory
+import java.util.*
+
+class ConsumerFactoryCustomizer : DefaultKafkaConsumerFactoryCustomizer {
+
+    override fun customize(consumerFactory: DefaultKafkaConsumerFactory<*, *>) {
+        if (Objects.nonNull(consumerFactory)) {
```

Comment:
To me, this is a kotlin anti-pattern. I think using kotlin's built in nullability checking is superior to using Java's Objects.notNull.